src/ngraph/runtime/backend.cpp

```diff
@@ -25,6 +31,28 @@
 using namespace std;
 using namespace ngraph;
 
+std::string runtime::Backend::s_backend_shared_library_search_directory;
+
+// This finds the full path of the containing shared library
+static string find_my_file()
+{
+#ifdef _WIN32
+    HMODULE hModule = GetModuleHandleW(L"ngraph.dll");
```

Comment:
The Windows implementation looks for ngraph.dll whereas the Linux/Darwin doesn't. Is that expected?